roles/sshd/tasks/main.yml

```diff
@@ -33,14 +34,22 @@
             itemPath: "vaults/{{ vault }}/items/sshd-host-keys"
       tags:
         - sshd
-
+    
+    - name: Add dbildungsplattform chart repo
+      kubernetes.core.helm_repository:
+        name: dbildungsplattform
+        repo_url: https://dbildungsplattform.github.io/helm-charts-registry/
+      check_mode: false
+      changed_when: false
+      tags: sshd
+      
     - name: Deploy latest version of SSHD chart to update user keys on Cluster
       kubernetes.core.helm:
-        name: sshd
-        chart_ref: "{{ playbook_dir }}/../charts/sshd"
         kubeconfig: "{{ kubeconfig }}"
-        create_namespace: true
-        release_namespace: "{{ sshd_namespace }}"
+        name: {{ sshd_chart }}
+        chart_ref: "dbildungsplattform/{{ sshd_chart }}"
+        update_repo_cache: yes
+        chart_version: "{{ infra_sshd_helm_chart_version }}"
```

Comment:
i would prefer if the var names would have a similiar naming, so that it is infra_sshd_chart or sshd_helm_chart_version.